oaknational/Oak-Web-Application
# Fix/lesq 319/lessons plurality [LESQ-319]

## Description

- update `UnitListLessonCount` to display "lesson" when `lessonCount` is 1

## How to test

1. Go to https://deploy-preview-1927--oak-web-application.netlify.thenational.academy/teachers/programmes/maths-secondary-ks4-foundation/units
2. You should see "1 lesson" for number 6 and plural "lessons" for the others

## ACs
- [ ]  For a unit with 1 lesson we show the detail on the card as 1 lesson
- [ ]  For a unit with more than 1 lesson, we show the detail on the card as X lessons
